### PR TITLE
Editorial review of the Getting started tutorial and Toolbox

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-graphql-8.surge.sh

File: modules/ROOT/pages/getting-started/index.adoc [hunk 1]
```diff
@@ -121,36 +129,33 @@ console.log(`🚀 Server ready at ${url}`);
...
+The address http://localhost:4000/ is the default URL in which Apollo Server starts at.
 
-Click "Query your server" which will open the Sandbox.
+== Create nodes in the database
 
-image::first-mutation.png[title="First mutation"]
-
-At the moment your database is empty! To get some data in there, you can create a movie and an actor in that movie, all in one mutation. The mutation in the screenshot above can also be found below:
+. Visit http://localhost:4000/ in your web browser. 
+You should get redirected to the Sandbox:
```
Comment (mjfwebb) [inline modules/ROOT/pages/getting-started/index.adoc:152]:
```suggestion
You should get redirected to the Apollo Sandbox:
```
I'm not 100% sure if it's required to mention it is the Apollo Sandbox we're referring to, but it was specified before.

File: modules/ROOT/pages/getting-started/toolbox.adoc [hunk 2]
```diff
@@ -1,38 +1,39 @@
...
-Then query the Neo4j database using the autogenerated GraphQL queries and mutations from the `@neo4j/graphql` library.
+. Query the Neo4j database using the autogenerated GraphQL queries and mutations from the `@neo4j/graphql` library:
++
+image::toolbox-editor-view.png[]
 
-image::toolbox-editor-view-v2.png[title="Query editor view"]
+== Store type definitions as favorite
 
-Note that you can store your type definitions as favorites and that a range of settings are available to work with.
+You can store your type definitions as favorites and that a range of settings are available to work with.
```
Comment (mjfwebb) [inline modules/ROOT/pages/getting-started/toolbox.adoc:35]: This sentence seems wrong now, so it probably needs to be reworded.

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,81 +1,78 @@
...
-:toc-title: Contents
-//:front-cover-image: image::title-page.png[]
-:header-title: NEO4J GRAPHQL LIBRARY
-:title-page-background-image: image::title-page.png[]
 
 The Neo4j GraphQL Library is a highly flexible, low-code, open source JavaScript library that enables rapid API development for cross-platform and mobile applications by tapping into the power of connected data.
 
-With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have application data treated as a graph natively from the front-end all the way to storage, avoiding duplicate schema work and ensuring flawless integration between front-end and backend developers.
+With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have data treated as a graph natively from the front-end all the way to storage.
+This avoids duplicate schema work and ensures flawless integration between front-end and backend developers.
```
Comment (mjfwebb) [inline modules/ROOT/pages/index.adoc:8]: Super picky but, do you think we should be consistent with hyphenation? front-end and back-end, or frontend and backend?

File: modules/ROOT/pages/getting-started/toolbox.adoc (hunk 2 above)
Comment (lidiazuin) [reply modules/ROOT/pages/getting-started/toolbox.adoc:35]: I don't understand why removing "note that" makes the sentence wrong? Could you please suggest another way or should we revert to its original?

File: modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (lidiazuin) [reply modules/ROOT/pages/index.adoc:8]: No, it's a good catch. I'm gonna check with the rest of the team which one we are using officially. Thanks for flagging it!

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-graphql-8.surge.sh

**Review by mjfwebb — APPROVED** [review]
Looks great, thanks!

**Pull request merged**